dbt_subprojects/solana/models/_sector/dex/phantom_swapper/phantom_swapper_solana_fee_payments_raw.sql

```diff
@@ -0,0 +1,112 @@
+{{ config(
+    schema = 'phantom_swapper_solana',
+    alias = 'fee_payments_raw',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    incremental_strategy = 'merge',
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+    unique_key = ['block_month', 'tx_id', 'token_address']
+   )
+}}
+
+{% set query_start_date = '2025-04-01' %} -- Not the true start date but queries time out if I go back farther
```

Comment:
you can set this back to normal start date to prep for merge. CI may fail, but we have good test runs to refer to.